### ISSUE TITLE
Extension prevents moving window to upper workspace

### ISSUE THREAD

**Issue by eriolloan** [marcinjahn/gnome-peek-top-bar-on-fullscreen-extension#18]
I added another monitor to my setup, right above the monitor hosting my main workspace.

When the extension is active, I can't drag windows from the main monitor to the secondary one. I guess it has to do with the trigger area...

**Comment by marcinjahn** [comment]
Yep, I never tested or thought about vertical setups. I don't think I'd have time to do that myself, but if someone wants to take it, a PR would be welcome.

**Comment by eriolloan** [comment]
I uunderstand tour stance and I agree it's not necessarily the most widespread monitor setup.

Sadly I don't have the knowledge required to contribute. 
(I don't even get how you extension devs gather and make sens of the spotty and partially outdated docs on the topic, so thanks you for the extension)

Le 30 mai 2024 13:36:23 GMT+02:00, Marcin Jahn ***@***.***> a écrit :

**Comment by marcinjahn** [comment]
Could be, not sure if it’s easily doable with Barrier though.
